packages/lesswrong/components/users/UsersAccountMenu.jsx

```diff
@@ -7,6 +7,12 @@ import Button from '@material-ui/core/Button';
 import { withStyles } from '@material-ui/core/styles';
 
 const styles = theme => ({
+  root: {
```

Comment:
When you paste a class into multiple components because it was used multiple times, can you at least add a TODO to remind us to resolve that later in some fashion?

(this feels like the class of copy-paste thing you were warning me about last week, and seems better to either address or flag it when you have it loaded in your context rather than leave us to discover it later)

Reply:
So turns out, 3 out of 4 of these styles had no effect. Removed those, reducing it to just the marginTop, which is more reasonable to have duplicated.

(I agonized for a bit over copy-pasting this, decided it wasn't enough total styling to justify commonizing, and... stupidly skipped the step of refactoring it down to a minimal form first. Oops.)